benchmarks/python/embedding_ops.py

```diff
@@ -0,0 +1,173 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+import torch
+from functools import partial
+
+
+from .model_configs import configs
+
+
+# Note this computes the bandwidth for weight gradient if the entire computation is done in a
+# single kernel. We don't consider the specific segmentation happening yet, since we don't have
+# a good clue how it should be segmented at this moment.
+def embedding_weight_grad_iobytes(config, dtype):
+    n_elements = 0
+    # adding size of grad
+    n_elements += config.batch_size * config.seq_len * config.hidden_size
+    # adding size of index
+    n_elements += config.batch_size * config.seq_len
+    # adding size of output (weight.grad)
+    n_elements += config.vocab_size * config.hidden_size
+    # scale by dtype size
+    return n_elements * dtype.itemsize
+
+
+def hf_qwen2(dtype):
```

Comment:
Do we need the outer function?
We can directly use the classes inherited from transformer models?

Reply:
Good call. those are identical code shared among models. Let me update that.

Reply:
refactored that. Does it look better now?